### PR TITLE
[DS-166]: [Pyxis Storybook] Documentation for Modal

### PR DESCRIPTION
**Pull request by emanuelemacri**
Add storybook documentation for Modal component

https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-166

### CODE REVIEW

**Comment by Cooryd** [comment]
👋 Hi! I've raised a draft PR for the `Modal` component [here](https://github.com/primait/pyxis/pull/177). I've been working against the spec described in [DS-204](https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-204). 

It looks like the config options detailed in these storybook docs differ quite significantly to the way I've built the component. I think we need a discussion on which way we would like to see the component configured so the two PRs can be aligned.

**Comment by emanuelemacri** [comment]
> 👋 Hi! I've raised a draft PR for the `Modal` component [here](https://github.com/primait/pyxis/pull/177). I've been working against the spec described in [DS-204](https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-204).
> 
> It looks like the config options detailed in these storybook docs differ quite significantly to the way I've built the component. I think we need a discussion on which way we would like to see the component configured so the two PRs can be aligned.

Hi @Cooryd ! We'll catch up in the meeting later, but in the meantime: this PR is usefull only to add the basic modal documentation on Storybook.

The "Modal" component you see inside is just an example to show the examples correctly (all the other Stories also use this method, but the components inside are just drafts waiting for the real components).

Regarding the API and the structuring of the component, the approach we saw together in the last meeting is the correct one.

As a first step in your PR I would just do the component and later we can update this documentation with the real component.

Thanks, see you later!

**Review by lauratrapani — CHANGES_REQUESTED** [review]

File: packages/pyxis-react/src/stories/components/Modal/Modal.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+} as ComponentMeta<typeof Modal>;
+
+const mobileFirstViewport = { viewport: { defaultViewport: "xxsmall" } }
+
+export const Default = () => <Modal />
+Default.parameters = renderSourceAsHTML(Default());
+
+export const DefaultOnMobile = () => <Modal />
+DefaultOnMobile.parameters = {
+  ...mobileFirstViewport,
```
Comment (lauratrapani) [inline packages/pyxis-react/src/stories/components/Modal/Modal.stories.tsx:18]: A sto punto potremmo uniformare anche sui tooltip così abbiamo le storie coerenti tra loro

**Review by lauratrapani — COMMENTED** [review]
L'ultima cosa poi per me ok

**Pull request merged**